crates/node/src/lib.rs

```diff
@@ -39,8 +39,8 @@ pub mod metrics;
 pub mod node;
 /// Testing functions
 ///
-/// Gated through the `testing` feature.
-#[cfg(feature = "testing")]
+/// Available during tests or when the `testing` feature is enabled.
+#[cfg(any(test, feature = "testing"))]
```

Comment:
I think this could fail if any of the tests use something behind `testing` but you didn't run cargo with `--features testing` (eg `cargo test`), so maybe the test command should be be updated in the Makefile

Reply:
Tests passed so maybe the `testing` feature was not needed? Or I'm missing something

Reply:
so you might have reviewed the outdated version still - apologies for that